### ISSUE TITLE
Setting to allow scrollbars to ALWAYS be visible, if content + view size requires scrollbars

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#30250]
Starting a new issue based on [this comment](https://github-com/microsoft/vscode/-/48153-issuecomment-1675316513) and its [reply](https://github-com/microsoft/vscode/-/48153-issuecomment-1675373001)

`Pane` is used throughout this request as an alternative name to `View`, as in `Explorer View`.

- TLDR; Can we have a setting to make the scrollbars of every pane be visible and not disappear, if content requires scrollbars?
This should apply when applicable, similar to a web page. If there is no more vertical or horizontal content to show, then the scrollbar does not need to be visible/shown. If there is enough content that a scrollbar would be shown if the mouse moved, show the scrollbars.

If it is desirable to only enable a subset of panes, it could be a setting that takes an array of pane setting names/values for which to enable it. Perhaps `"*"` could represent enabling it for all.

A similar setting exists to `Autohide` the minimap, if it is enabled, and this is disabled by default.
> ⚠️ If you read through the linked issue thread above, another issue posters tried to fix was content running through the scrollbar area. This was averted by enabling the minimap; people who didn't want to use the minimap then set the width to 0. Perhaps the content running through could be fixed with this "always-enabled" scrollbar update, or that may have to be fixed before this one?

Basically, it seems like an accessibility issue, or at least a difference of expectations, when the scrollbars disappear upon navigating to new windows or editor panes. The "expectations," having used mostly Windows, web browsers, and many other apps, is that the scrollbar remains visible. I know that some operating systems, I think OS X and now Windows 11, hide the scrollbar if the mouse isn't moving, but at least Windows 11 (maybe OS X?) has a system setting to "Always show scrollbars", and this is under `Accessibility` -> `Visual effects`.
